src/components/Configure/content/fields/FieldMappings/ValuesFieldMapping.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { ComboBox } from 'src/components/ui-base/ComboBox/ComboBox';
+
+import { useSelectedConfigureState } from '../../useSelectedConfigureState';
+
+interface ValuesFieldMappingProps {
+  value: any,
+  fieldName: any,
+  onSelectChange: (
+    e: React.ChangeEvent<HTMLSelectElement>
+  ) => void,
+  allValues: any[],
+}
+
+export function ValuesFieldMapping(
```

Comment:
The presence of `Field` in the name of this component is a little unintuitive to me, maybe `ValueMappingItem` or something like that?